pages/api/books/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
We can actually leverage Prisma directly for these endpoints! 😄 

https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest

Reply:
I'll check this out! I'll probably merge this one in and create a branch for a refactor!